process_request/clients.py

```diff
@@ -0,0 +1,35 @@
+from request_broker import settings
+from requests import Session
+
+
+def http_meth_factory(meth):
+    """Utility method for producing HTTP proxy methods.
+
+    Urls are prefixed with the value of baseurl from the client's config.
+    Arguments are passed unaltered to the matching requests.Session method."""
```

Comment:
```suggestion
    Arguments are passed unaltered to the matching requests. Session method."""
```

Reply:
This is actually right - `requests.Session` refers to a `requests` class called `Session`.